djautotask/models.py

```diff
@@ -52,13 +71,103 @@ class Picklist(TimeStampedModel):
     class Meta:
         abstract = True
 
+    def __str__(self):
+        return '{}-{}'.format(self.value, self.label)
+
 
 class TicketStatus(Picklist):
     pass
 
     class Meta:
-        verbose_name = 'Ticket status'
         verbose_name_plural = 'Ticket statuses'
 
+
+class TicketPriority(Picklist):
+    pass
+
+    class Meta:
+        verbose_name_plural = 'Ticket priorities'
+
+
+class Queue(Picklist):
+    pass
+
+
+class ProjectStatus(Picklist):
+    pass
+
+    class Meta:
+        verbose_name_plural = 'Project statuses'
+
+
+class ProjectType(Picklist):
```

Comment:
Now this and status were not technically part of this issue, but I can't think of a good enough reason to say take it out at this point... so :woman_shrugging:

Reply:
Yeah I know. I'm guilty. I figured they'll be useful later on and they were super easy to add while I was doing everything else.